pkgs/applications/networking/irc/weechat/scripts/multiline/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchurl, substituteAll, PodParser }:
+
+stdenv.mkDerivation {
+  pname = "multiline";
+  version = "0.6.3";
+
+  src = fetchurl {
+    url = "https://raw.githubusercontent.com/weechat/scripts/945315bed4bc2beaf1e47f9b946ffe8f638f77fe/perl/multiline.pl";
+    sha256 = "1smialb21ny7brhij4sbw46xvsmrdv6ig2da0ip63ga2afngwsy4";
+  };
+
+  dontUnpack = true;
+  prePatch = "cp $src multiline.pl";
```

Comment:
```suggestion
  prePatch = ''
    cp $src/perl/multiline.pl multiline.pl
  '';
```

Reply:
Just out of curiosity, since when should single commands be within a multi-line string?

Reply:
Fixed, but I'd love to have an answer.

Reply:
Because we had multiple occasions where commands where not concatenated correctly which could be avoided by this and it makes it easier to add new commands.

Reply:
That makes sense, thanks.